pkg/distro/rhel8/images.go

```diff
@@ -331,6 +331,11 @@ func imageInstallerImage(workload workload.Workload,
 	img.AdditionalDracutModules = []string{"prefixdevname", "prefixdevname-tools"}
 	img.AdditionalAnacondaModules = []string{"org.fedoraproject.Anaconda.Modules.Users"}
 
+	if instCust := customizations.GetInstaller(); instCust != nil {
```

Comment:
This is a wider issue with how image functions initialise and configure their pipelines.  The way it's designed just makes for a lot of repetition when the same pipeline is used.  I agree it would be great to have a single place for doing it but it's something that would need more work.